### DISCUSSION TITLE
Aligning two scans before +append

### DISCUSSION THREAD

**Discussion by smerfmaruda** [ImageMagick/ImageMagick#4631] [Help] [answered]
Cheers. I have scans of two pages that I want to `+append` together. The problem is the scans are not perfectly aligned: they show a picture and a `+append` with no adjustments show a few pixel offset between two which, needlessly to say, does not look nice.

How do I align these two? I am fine with guessing the offset i.e. I may try out few values for the best result but what command should I use to "move up" one or "move down" the other scan?

**Comment by fmw42** [comment, marked as answer]
You can use -smush X  to append top to bottom or +smush X to append left to right with a gap or overlap of X pixels. See https://imagemagick.org/script/command-line-options.php#smush

**Reply by smerfmaruda** [reply]
Aw, long forgotten sweet `smush`, yes. Problem with IM is that if you don't use it on a daily basis, you just tend to forget all the commands / tricks you once knew. Me at least.

Thanks, that's what I've been looking for. For a quick and dirty solution I added a top border to one of the pages but `smush` may be just more elegant.

**Reply by Jmuccigr** [reply]
> Problem with IM is that if you don't use it on a daily basis, you just tend to forget all the commands / tricks you once knew. Me at least.

Ha, me too! That's why I added a reference to smush into the docs for append and vice versa. Good to have the docs give you a hint. :-)
